openqa-investigate

```diff
@@ -243,12 +243,12 @@ main() {
     fi
     set -u
     clone_call="${clone_call:-"$client_prefix openqa-clone-job --skip-chained-deps --max-depth 0 --parental-inheritance --within-instance"}"
-    error_count=0
+    local rc=0
     # shellcheck disable=SC2013
     for i in $(cat - | sed 's/ .*$//'); do
-        investigate "$i" "$@" || ((error_count++)) ||:
+        investigate "$i" "$@" || rc=$?
```

Comment:
why did we have an error count in the first place when you are resorting to the simpler solution now?

Reply:
Ask that @okurz from two years ago: https://github.com/os-autoinst/scripts/commit/60a8c4c80ec4ec8ca956a0782ea78d9ccd375d45

My simpler code has the caveat to only return the return code of the last investigate call. Maybe you wanted to avoid that. However, I don't think we use the count somewhere. Note that the count was broken anyways for more than 255 errors.

Reply:
Well, the subject line "Prevent immediate failure if just a single job fails" already explains that, thank you @okurz from two years ago ;) So why should we exit immediately now on the first failure?

Reply:
huh, weird. Ok then :)